### PR TITLE
Fix for rendering of code blocks in CLE documentation

### PR DESCRIPTION
**Pull request by absurdfarce**
Looks like I wasn't following rst as closely as I should've been.

Relevant notes on code block formatting can be found [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives).

### CODE REVIEW

File: docs/column_encryption.rst [hunk 1]
```diff
@@ -57,6 +58,7 @@ supplied parameters.  For example, we can create a prepared statement to insert
 by executing the following code after creating a :class:`~.Cluster` in the manner described above:
 
 .. code-block:: python
+
```
Comment (jgillenwater) [inline docs/column_encryption.rst:61]:
```suggestion
by executing the following code after creating a :class:`~.Cluster` in the manner described above::

```

File: docs/column_encryption.rst [hunk 2]
```diff
@@ -70,6 +72,7 @@ standard serialization methods employed by the driver.  The result is then encry
 of the policy.  Using this approach the example above could be implemented along the lines of the following:
 
 .. code-block:: python
+
```
Comment (jgillenwater) [inline docs/column_encryption.rst:75]:
```suggestion
of the policy.  Using this approach the example above could be implemented along the lines of the following::

```

**Review by jgillenwater — CHANGES_REQUESTED** [review]
I left two comments that would make this structured like some of the codeblocks in other topics. Can we try this instead?

**Comment by absurdfarce** [comment]
Met with @jgillenwater to discuss and we settled on the Sphinx-specific code-block usage for now.  Also filed PYTHON-1349 to come up with a more robust long-term answer for this across the entire set of Python driver docs.

**Pull request merged**